plugins/linuxplugin/l3plugin/model/l3/l3.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package l3;
+
+enum Scope {
+    GLOBAL = 0;
+    SITE = 1;
+    LINK = 2;
+    HOST = 3;
+};
+
+/* static ip routes */
+message LinuxStaticRoutes {
+    message Route {
+        string interface = 1;       /* interface name */
```

Comment:
Both Route and ArpEntry will also need:
- name (unique across namespaces)
- namespace: see message Namespace in interfaces.proto - it would be ideal if we could reuse the definition and have it on one place only using import (https://developers.google.com/protocol-buffers/docs/proto#other)